README.md

```diff
@@ -5,6 +5,25 @@ description: >-
 ---
 
 # Introduction
+## OpenLab
+
+OpenLab is a community-led program to test and improve support for the most popular Software Development Kits(SDKs) as well as platforms like Kubernetes, Terraform, CloudFoundry and more. The goal is to improve the usability, reliability and resiliency of tools and applications for hybrid and multi-cloud environments.
+
+## How to Join OpenLab
+
+There is plenty of room for helping the shape of future of OpenLab and ensuring things run smoothly. Welcome to join OpenLab if you're interested in our system, please follow the guide below:
+
+### To be an OpenLab member
+Join us to be a member of OpenLab firstly, then you will find out what's happening in OpenLab, what fellow open source community members are doing using OpenLab resources, also as a member you can access OpenLab resources for your testing, please follow the simple [member application](Member_Application.md) guide.
+
+### Test Request
+If you wish to use OpenLab resources, start your test request [here](./contributor/Testing_Request.md).
```

Comment:
If you have an idea or real use case about some integration scenarios, like: integrating Kubernetes and OpenStack, plan to do proof of concept(POC) against some technologies, all you have to do is just sharing your idea and case in OpenLab, if others are interested on your case, they maybe pick up your task and complete the subsequent testing works, pay attention on updating of OpenLab [web site](https://openlabtesting.org/explore/), we will share the test result on it. Start your test request [here](./contributor/Testing_Request.md) now.